src/feature/rustc.rs

```diff
@@ -150,15 +150,15 @@ pub(crate) fn configure_rustc(instructions: Instructions, config: &mut Config) -
             add_entry(
                 config.cfg_map_mut(),
                 VergenKey::RustcCommitHash,
-                rustc.commit_hash,
+                Some(rustc.commit_hash.unwrap_or("unknown".to_string())),
```

Comment:
This is failing the lint checks.   Can you update the `unwrap_or` to `unwrap_or_else(|| "unknown".to_string())`

Reply:
Woops, sorry, forgot to run clippy :sweat_smile: